src/lib/util/cdrom.cpp

```diff
@@ -80,6 +80,9 @@ const int SYNC_NUM_BYTES = 12;
 /** @brief  offset within sector. */
 const int MODE_OFFSET = 0x00f;
 
+/** @brief  EDC offsets */
+const int EDC_MODE1_OFFSET = 16+2048;
```

Comment:
Please make this and the other constants here `constexpr` to avoid leaking symbols.

Reply:
resolved in 9faaf3246d4f0753828e155073ff9dc9127b892b